tensorflow/python/compiler/tensorrt/__init__.py

```diff
@@ -15,5 +15,65 @@
 """Exposes the python wrapper for TensorRT graph transforms."""
 
 # pylint: disable=unused-import,line-too-long
+from tensorflow.compiler.tf2tensorrt import _pywrap_py_utils
 from tensorflow.python.compiler.tensorrt import trt_convert as trt
+from tensorflow.python.compiler.tensorrt import utils as trt_utils
+from tensorflow.python.platform import tf_logging as logging
 # pylint: enable=unused-import,line-too-long
+
+
+if not _pywrap_py_utils.is_tensorrt_enabled():
+  logging.error(
+      "Tensorflow needs to be built with TensorRT support enabled to allow "
+      "TF-TRT to operate.")
+
+  raise RuntimeError("Tensorflow has not been built with TensorRT support.")
+
+else:
+  linked_version = _pywrap_py_utils.get_linked_tensorrt_version()
+  loaded_version = _pywrap_py_utils.get_loaded_tensorrt_version()
+
+  logging.info("Linked TensorRT version: %s", str(linked_version))
+  logging.info("Loaded TensorRT version: %s", str(loaded_version))
+
+
+  def raise_trt_version_deprecated(version_type, trt_version):
```

Comment:
Let's rename the routine to match what it does, prefix it with _ to make it private in the module, unnest it. We mentioned the style guide for nested function in another PR recently.
   _raise_trt_version_no_supported

Reply:
Fixed